doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md

```diff
@@ -28,7 +28,16 @@ YouTube Video
 YouTube Video
 {% endembed %}
 
-Some other set 1 learning videos are not available yet. Check back on the 9th of October 2022.
+### Create, edit and delete files or folders using GITHUB UI)
+{% embed url="https://youtu.be/lqSdpFmT3y4" %}
+YouTube Video
+{% endembed %}
+
+### When to delete a branch connected to pull request or a fork
+{% embed url="https://youtu.be/K8PmM6mRLIo" %}
+YouTube Video
+{% endembed %}
```

Comment:
Hi @rk-1620
Nice work finding the right file and adding the changes in the right place.

But I cannot accept this change. The reason is that it is not advisable to make changes directly in the develop branch. There are also other things written in our guide that you didn't take note of. Just close this pull request and delete your fork and start all over.

This time, please follow the guidelines here for your new pull request to be accepted: https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/making-and-submitting-changes.

If you have any questions or are confused about anything feel free to ask me. I will gladly respond.

Reply:
@Ifycode Thank you for your guidance. I will make changes to it and make a proper and new pull request.

Reply:
🙂👍